### DISCUSSION TITLE
Bearer Token authenticated

### DISCUSSION THREAD

**Discussion by zhanghj7** [ory/hydra#2501] [Q&A] [answered]
Self-generated Bearer Token authenticated in ory hydra，
My own system has generated the token,  how can I authenticate it in the Ory Hydra？
Can I get through？

**Comment by vinckr** [comment, marked as answer]
Sorry you're having trouble and the late reply. 

We'll need more information in order to figure out what's happening here.
Can you provide some additional information, such as what were you trying to do when the error occurred?
Your setup, additional log files, the software version, your configuration and any other information that you think is helpful also helps us to solve your issue more quickly!

Thanks!

**Reply by vinckr** [reply]
Hey @zhanghj7, just checking in if your problem still persists. 
I will mark this as answered for now, but feel free to come back to the discussion any time.
Thanks a ton :bee:
